subprojects/common/kotlin-ast-parser/src/main/kotlin/com/avito/instrumentation/impact/KotlinClassesFinder.kt

```diff
@@ -0,0 +1,10 @@
+package com.avito.instrumentation.impact
+
+import java.io.File
+
+interface KotlinClassesFinder {
+
+    fun find(file: File): List<String>
```

Comment:
// todo describe return value

Reply:
typealias would be good

Reply:
Made a FullClassName model, because we need separate access to package and shortclassname in different consumers